src/components/common/Dropdown/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./style.scss";
+
+export default ({ isOpen, className, options = [] }) => {
+  if (isOpen) {
+    return (
+      <div className={`dropdown ${className ? className : ""}`}>
+        {options.map((option) => {
+          return (
+            <button
+              className={`dropdown-button ${
+                option.className ? option.className : ""
+              }`}
+              key={option.value}
+              onClick={option.onClick}
+            >
+              {option.label}
+            </button>
+          );
+        })}
+      </div>
+    );
+  }
+  return null;
```

Comment:
Is returning `null` from a  component a good practice ?
Unaware about this.

Reply:
Yes, if we don't want to render anything then returning null is fine.